docs/content/examples/uses-cases/folders.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Use Cases | Folders'
+hide:
+  - toc # Hide Table of Contents for this page
+---
+## Introduction
+
+This provides a solution letting different email clients sharing the same archive folder.
+It might also help when troubleshooting sharing `Sent` and `Drafts` folders. 
+
+## Adding a mailbox folder
+
+See [`target/dovecot/15-mailboxes.conf`][gh-config-dovecot-mailboxes] for existing folder definitions you can modify or enable.
+Dovecot also [provides their own example config][dovecot-config-mailboxes] for reference, with further information in [their documentation][dovecot-docs-mailboxes].
+
+## Caution
+
+### Adding folders to an existing setup
+
+Handling of newly added mailbox folders can be inconsistent across mail clients:
+
+- Users may experience issues such as archived emails only being available locally.
+- Users may need to migrate emails manually between two folders.
+
+### Special Use Mailbox Support
+
+Not all mail clients support Special Use Mailbox (_defined in [RFC 6154][rfc-6154]_), using the exact mailbox name instead. To support those clients, you may prefer to keep the tag and folder name identical.
+
+### Internationalization (i18n)
+
+Users and mail clients may prefer localized mailbox names instead of English. Take care to test localized names work well, keep in mind concerns such as tag support.
```

Comment:
As you are making commits and resolving prior review with no discussion, "tag" is presumably still unclear? Please correct the naming or provide context what a "tag" refers to in the documentation.

I am placing trust in your knowledge on the topic or other reviewers here. My only involvement is QA for docs.

Reply:
@polarathene I think I removed the reference of "tag" everywhere. And now it should only use "Special Use Mailbox" which is the term in RFC. I saw this "tag" usage when I search for the solution for this issue. But yes, it may cause confusion.

Reply:
"tag" is still referenced in the current document:

> _To support those clients, you may prefer to keep the tag and folder name identical._

> _keep in mind concerns such as tag support._

If "tag" is a common way to talk about it, like "folder" for mailbox, then just hint that to the reader. eg: `### Special Use Mailbox (_aka Tag_) Support`?

If you do that, then it is clear what "tag" refers to in the quotes to someone who may not know any of this from anywhere else, just reading the example page you are contributing and interested in the feature.

Reply:
Updated PR with "tags" references removed. It's much less common than "folder".